src/models/items.py

```
@@ -25,23 +28,24 @@ class Item:
     rarity: ItemRarity
     drop_chance: float = 0.1
 
-    def use(self, target: 'Character') -> bool:
+    def use(self, target: "Character") -> bool:
```

Ruff (0.8.0)

Undefined name `Character`

(F821)